static/src/components/CardHist.js

```diff
@@ -1,48 +1,88 @@
 import React, { PropTypes } from 'react';
-import { Card, CardText, CardHeader, CardMedia, CardTitle, GridList } from 'material-ui';
+import { CardMedia, Table, TableBody, TableRow, TableRowColumn } from 'material-ui';
+
+function sortByColor(props) {
+    const unsorted = props;
+
+    let brown = [];
+    let grey = [];
+    let yellow = [];
+    let blue = [];
+    let green = [];
+    let red = [];
+
+    for (let i = 1; i < unsorted.length; ++i) {
```

Comment:
Does the index of history start at 1?

Reply:
Starts at 1 because index zero is the wonder board itself. No need to display that, just the cards